Assignment4/Assignment4.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+
+function checkError(error){
+  if(error)
+    throw error;
+}
+
+function print(string)
+{
+  console.log(string);
+}
+
+function readAppendWrite(readFile, appendFile, writeFile) {
+  fs.readFile (readFile, function (errorOnFile1Read, dataFromFile1) {
```

Comment:
No Space after function calling and before its argument. This should be like `fs.readFile(read......`
Problem Type: Spacing